src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfSource = sourceString
+    .split(';')
+    .filter((part) => part.trim() !== '');
```

Comment:
You can chain the 'map' method directly after the 'filter' method to improve readability. Ensure that each method call starts on a new line with the same indentation level.